app/main.py

```diff
@@ -33,23 +33,23 @@
     level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s"
 )
 
-
 class MainApp:
     def __init__(self):
         load_dotenv(override=True)
+        if os.getenv("isDevelopmentMode") == "enabled":
```

Comment:
correct condition but readability can be improved by using and instead of inner if

Reply:
this was an embarrassing mistake, my bad